common/tests/fixtures/ethereum/getaddress.json

```diff
@@ -6,31 +6,31 @@
     "tests": [
         {
             "parameters": {
-                "path": "m/44'/60'/0'"
+                "path": "44'/60'/0'"
```

Comment:
Please modify this the other way.

Having the path start with `m/` makes it easier to auto-detect that it's a derivation path, and Cardano fixtures also use `m/` everywhere.

Reply:
Fixed in 68e03e7